json_serializable/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - Support properties where the getter is defined in a class with a corresponding
   setter in a super type.
+- `JsonKey.unknownEnumValue`
```

Comment:
Please move this to a new entry under `3.4.1-dev` – also update the pubspec to have that version `3.4.1-dev`

Reply:
Or `3.4.2-dev` since you're already getting ready for the `3.4.1` release on `master`?

Reply:
Set it to `3.4.1-dev` as requested. Please let me know if this was a typo and you meant `3.4.2-dev` instead.